android/src/main/java/com/stripeterminalreactnative/ReactNativeConstants.kt

```diff
@@ -13,4 +13,7 @@ enum class ReactNativeConstants(val listenerName: String) {
     REPORT_UPDATE_PROGRESS("didReportReaderSoftwareUpdateProgress"),
     START_INSTALLING_UPDATE("didStartInstallingUpdate"),
     UPDATE_DISCOVERED_READERS("didUpdateDiscoveredReaders"),
+    START_READER_RECONNECT("didStartReaderReconnect"),
+    TERMINAL_SUCCEED_READER_RECONNECT("terminalDidSucceedReaderReconnect"),
+    TERMINAL_FAIL_READER_RECONNECT("terminalDidFailReaderReconnect"),
```

Comment:
I think we should drop TERMINAL from these. "Terminal" is implied (for the native iOS callbacks it was added for consistency with other Terminal delegates but in a future version of the SDK its switching to be a reader delegate and will include the instance of the reader that succeeded or failed).

```
    SUCCEED_READER_RECONNECT("didSucceedReaderReconnect"),
    FAIL_READER_RECONNECT("didFailReaderReconnect"),
```